app/controllers/bikes/recovery_controller.rb

```diff
@@ -16,7 +16,8 @@ def update
         flash[:success] = translation(:bike_recovered)
         redirect_to bike_path(@bike)
       else
-        render :edit, bike_id: @bike.id, token: params[:token]
+        session[:recovery_link_token] = params[:token]
```

Comment:
Hot take: SICK! Good addition, makes life better for everyone.

Reply:
Oh. This was something that should've been there already. Oops, my bad. Any way you can add a test?